src/main/java/dbaccessor/message/MessageAccessor.java

```diff
@@ -42,9 +51,21 @@ public void updateReceivedTs(Message message) {
     /**
      * Completes all DB insertions for the CreateMessage API in a single transaction.
```

Comment:
Missed out in the previous review, what does the `boolean` returned value of this method signify. Please document that.

Reply:
Done

Reply:
Though I've added documentation for the significance of the returned value, honestly speaking, we don't really need to return anything as per the method use. But it was mandatory to return something from the `transaction` 
(i.e. `performReadWriteTransaction`), hence I returned `true` for success.

Reply:
Testing for failures can be done in a fake (or in-memory) DB easily.
Not sure how we can do that with cloud spanner which doesn't provide such an in-memory db.